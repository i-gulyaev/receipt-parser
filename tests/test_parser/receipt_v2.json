[{"status":2,"seller":{"name":"ООО \"Лента\"","inn":"7814148471"},"operation":{"date":676724700,"type":1,"sum":139430},"id":"62a5dc79f59138016473d166","organization":{"name":"ООО \"Лента\"","inn":"7814148471"},"qr":"t=20220612T1405&s=1394.30&fn=9960440302673064&i=351&fp=3001468322&n=1","query":{"date":"2022-06-12T14:05","documentId":351,"fsId":"9960440302673064","fiscalSign":"3001468322","operationType":1,"sum":139430},"ticket":{"document":{"receipt":{"retailPlace":"ТК Лента-734","totalSum":139430,"userInn":"7814148471  ","code":3,"items":[{"ndsSum":1774,"quantity":1.3320000000000001,"productType":1,"nds":1,"price":7989,"paymentType":4,"name":"Бананы вес 1кг","sum":10641},{"ndsSum":1235,"quantity":0.77200000000000002,"productType":1,"nds":2,"price":17599,"paymentType":4,"name":"Яблоки Гольден вес 1кг","sum":13586},{"ndsSum":1036,"quantity":1,"productType":1,"nds":2,"price":11399,"paymentType":4,"name":"Абрикосы импорт 500г","sum":11399},{"ndsSum":3783,"quantity":1,"productType":1,"nds":1,"price":22699,"paymentType":4,"name":"Курага ЛЕНТА без косточки 200г","sum":22699},{"ndsSum":5950,"quantity":1,"productType":1,"nds":1,"price":35699,"paymentType":4,"name":"Кешью DOLCE ALBERO жареный 200г","sum":35699},{"ndsSum":946,"quantity":0.30599999999999999,"productType":1,"nds":2,"price":33989,"paymentType":4,"name":"Черешня вес 1кг","sum":10401},{"ndsSum":1521,"quantity":1,"productType":1,"nds":2,"price":16729,"paymentType":4,"name":"Сыр MILKANA GrandBlu Интенс плес 50% 100","sum":16729},{"ndsSum":1661,"quantity":1,"productType":1,"nds":2,"price":18276,"paymentType":4,"name":"Мороженое КОРОВ ИЗ КОРЕНОВ пл пол15%400г","sum":18276}],"ecashTotalSum":139430,"fiscalDriveNumber":"9960440302673064","shiftNumber":4,"nds18":11507,"kktRegId":"0005473019015991    ","provisionSum":0,"dateTime":1655031900,"requestNumber":19,"retailPlaceAddress":"Россия, 197350, г. Санкт-Петербург, Парашютная ул., д. 61, к. 4, СТР. 1, помещ.","fiscalSign":3001468322,"user":"ООО \"Лента\"","prepaidSum":0,"cashTotalSum":0,"buyerPhoneOrAddress":"none","fiscalDocumentNumber":351,"operationType":1,"fiscalDocumentFormatVer":4,"creditSum":0,"taxationType":1,"nds10":6399}}},"kind":"kkt","createdAt":676729850}]