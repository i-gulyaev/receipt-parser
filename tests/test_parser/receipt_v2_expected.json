{
  "seller": "ООО \"Лента\"",
  "sum": 1394.3,
  "date": "2022-06-12T14:05:00",
  "items": [
    {
      "quantity": 1.332,
      "price": 79.89,
      "name": "Бананы вес 1кг",
      "sum": 106.41
    },
    {
      "quantity": 0.772,
      "price": 17599,
      "name": "Яблоки Гольден вес 1кг",
      "sum": 135.86
    },
    {
      "quantity": 1,
      "price": 113.99,
      "name": "Абрикосы импорт 500г",
      "sum": 113.99
    },
    {
      "quantity": 1,
      "price": 226.99,
      "name": "Курага ЛЕНТА без косточки 200г",
      "sum": 226.99
    },
    {
      "quantity": 1,
      "price": 356.99,
      "name": "Кешью DOLCE ALBERO жареный 200г",
      "sum": 356.99
    },
    {
      "quantity": 0.306,
      "price": 339.89,
      "name": "Черешня вес 1кг",
      "sum": 104.01
    },
    {
      "quantity": 1,
      "price": 167.29,
      "name": "Сыр MILKANA GrandBlu Интенс плес 50% 100",
      "sum": 167.29
    },
    {
      "quantity": 1,
      "price": 182.76,
      "name": "Мороженое КОРОВ ИЗ КОРЕНОВ пл пол15%400г",
      "sum": 182.76
    }
  ]
}
